spec/manageiq/journald_spec.rb

```diff
@@ -9,15 +7,29 @@
     end
   end
 
-  context "syslog_identifier accessor" do
-    it "has a syslog_identifier accessor" do
-      expect(logger).to respond_to(:syslog_identifier)
-      expect(logger).to respond_to(:syslog_identifier=)
+  context "deprecated syslog_identifier accessor" do
+    it "sets the progname" do
+      logger.syslog_identifier = "manageiq-test"
+      expect(logger.syslog_identifier).to eq('manageiq-test')
+      expect(logger.progname).to          eq('manageiq-test')
+    end
+
+    it "is set by the progname" do
+      logger.progname = "manageiq-test"
+      expect(logger.syslog_identifier).to eq('manageiq-test')
+      expect(logger.progname).to          eq('manageiq-test')
+    end
+
+    it "sets the progname on instantiation" do
+      logger = described_class.new(nil, :syslog_identifier => 'manageiq-test')
```

Comment:
It doesn't look like `ManageIQ::Loggers::Journald` ever accepted `:syslog_identifier` as an argument: https://github.com/ManageIQ/manageiq-loggers/pull/11/files#diff-b742abd20b3760b523f7f2c043cbd4e158ec8ffa839b6ccff3c059b7d4bcd895R18

Reply:
oh yeah you are right - weird that this all went green locally.

Reply:
Updated